presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -4686,7 +4686,7 @@ public void testAccessControl()
     {
         assertAccessDenied("INSERT INTO orders SELECT * FROM orders", "Cannot insert into table .*.orders.*", privilege("orders", INSERT_TABLE));
         assertAccessDenied("DELETE FROM orders", "Cannot delete from table .*.orders.*", privilege("orders", DELETE_TABLE));
-        assertAccessDenied("CREATE TABLE foo AS SELECT * FROM orders", "Cannot create table .*.foo.*", privilege("foo", CREATE_TABLE));
+        assertAccessDenied("CREATE TABLE create_access_denied AS SELECT * FROM orders", "Cannot create table .*.create_access_denied.*", privilege("create_access_denied", CREATE_TABLE));
```

Comment:
the issue here is that this table is not actually created. So there is other table that was created and not removed. So you are fixing symptoms not a cause.

Reply:
There is another test: `testDescribeOutputNonSelect` which uses `foo` table too.

Reply:
That test also does not create a table.

Reply:
I think it does